packages/frontend/editor-ui/src/components/canvas/Canvas.vue

```diff
@@ -541,10 +548,16 @@ const isPaneMoving = ref(false);
 
 useViewportAutoAdjust(viewportRef, viewport, setViewport);
 
-function getProjectedPosition(event?: Pick<MouseEvent, 'clientX' | 'clientY'>) {
+function getProjectedPosition(
+	event?: Pick<MouseEvent, 'clientX' | 'clientY'> | Pick<TouchEvent, 'touches'>,
+) {
 	const bounds = viewportRef.value?.getBoundingClientRect() ?? { left: 0, top: 0 };
-	const offsetX = event?.clientX ?? 0;
-	const offsetY = event?.clientY ?? 0;
+
+	const clientX = event && 'clientX' in event ? event.clientX : event?.touches?.[0]?.clientX;
```

Comment:
There is a helper for this `getMousePosition` in `nodeViewUtils`, although it uses pageX/Y instead of clientX/Y